arches_lingo/src/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue

```diff
@@ -0,0 +1,195 @@
+<script setup lang="ts">
+import { inject, ref, onMounted } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+
+import FileListWidget from "@/arches_component_lab/widgets/FileListWidget/FileListWidget.vue";
+
+import { DANGER, SECONDARY, VIEW } from "@/arches_lingo/constants.ts";
+import { useConfirm } from "primevue/useconfirm";
+
+import type {
+    ConceptImages,
+    DigitalObjectInstance,
+} from "@/arches_lingo/types.ts";
+import { fetchLingoResource } from "@/arches_lingo/api.ts";
+import NonLocalizedStringWidget from "@/arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget.vue";
+
+const props = defineProps<{
+    tileData: ConceptImages | undefined;
+    componentName: string;
+    sectionTitle: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+}>();
+
+const resources = ref<DigitalObjectInstance[]>();
+const { $gettext } = useGettext();
+const confirm = useConfirm();
+
+onMounted(async () => {
+    if (props.tileData) {
+        resources.value = await Promise.all(
+            props.tileData.aliased_data.depicting_digital_asset_internal.map(
+                async (digitalAsset) =>
+                    await getConceptImageResource(digitalAsset.resourceId),
+            ),
+        );
```

Comment:
hmm. this has the potential to spam the server. Would it be possible to collect the resourceIds, make a single call, then map the values?

Reply:
@jacobtylerwalls is there an existing pattern to batch known resources?

Reply:
Yep, there's a comma-separated `resource_ids` [query param](https://github.com/archesproject/arches-querysets/blob/cfa473ab5418b67421c7486233c0e7715aaca117/arches_querysets/rest_framework/view_mixins.py#L45).